tests/test_relation.py

```diff
@@ -98,6 +101,25 @@ def test_insert_select(self):
             'real_id', 'date_of_birth', 'subject_notes', subject_id='subject_id+1000', species='"human"'))
         assert_equal(len(self.subject), 2*original_length)
 
+    @raises(InternalError)
```

Comment:
did notice & agree this might be better - will need to xref against 'normal' queries without ignore_extra_fields as well since it doesn't look like this exception is caught in that case. Indeed other mysql exceptions are also thrown elsewhere so perhaps this should be a separate issue?